crates/libs/result/src/hresult.rs

```diff
@@ -80,7 +79,7 @@ impl HRESULT {
                     flags |= FORMAT_MESSAGE_FROM_HMODULE;
 
                     module = LoadLibraryExA(
-                        b"ntdll.dll\0".as_ptr(),
+                        c"ntdll.dll".as_ptr() as _,
```

Comment:
This is a false positive as Clippy insists I use c"..." but that fails to compile since the result is a signed pointer whereas `LoadLibraryExA` expects an unsigned pointer...